src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_Border.cs

```diff
@@ -102,5 +110,222 @@ public async Task When_Border_Centered_With_Margin_Inside_Wide_Rectangle()
+			}
+
+			var SUT = new Border_CornerRadius();
+			var sample = SUT.Sample1;
+
+			float eighth = (float)sample.Width / 8;
+			WindowHelper.WindowContent = SUT;
+			WindowHelper.WaitForLoaded(SUT);
+			var renderer = new RenderTargetBitmap();
+			await WindowHelper.WaitForIdle();
+			await renderer.RenderAsync(SUT);
+			var result = await RawBitmap.From(renderer, SUT);
+			WindowHelper.WaitForIdle();
+
+			ImageAssert.HasColorAtChild(result, sample, eighth, (float)sample.Height - eighth, noColor, tolerance: 25);
+			ImageAssert.HasColorAtChild(result, sample, (float)sample.Width - eighth, (float)sample.Height - eighth, noColor, tolerance: 25);
+			ImageAssert.HasColorAtChild(result, sample, (float)sample.Width - eighth, eighth, noColor, tolerance: 25);
+			ImageAssert.HasColorAtChild(result, sample, eighth, eighth, noColor, tolerance: 25);
+			ImageAssert.HasColorAtChild(result, sample, (float)sample.Width / 2, (float)sample.Height / 2, reddish, tolerance: 25);		
+		}
+
+		[TestMethod]
+		public async Task When_Border_CornerRadius_BorderThickness()
+		{
+			//Same colors but with the addition of a White background color underneath
+			const string lightPink = "#FF7F7F";
+			const string lightBlue = "#7F7FFF";
+
+			if (!ApiInformation.IsTypePresent("Windows.UI.Xaml.Media.Imaging.RenderTargetBitmap"))
+			{
+				Assert.Inconclusive(); // System.NotImplementedException: RenderTargetBitmap is not supported on this platform.;
+			}
+
+			var expectedColors = new[]
+			{
+				new ExpectedColor { Thicknesses = new [] { 10, 10, 10, 10 }, Colors = new [] { lightPink, lightPink, lightPink, lightPink } },
+				new ExpectedColor { Thicknesses = new [] { 10, 0, 10, 10 }, Colors = new [] { lightPink, lightBlue, lightPink, lightPink } },
+				new ExpectedColor { Thicknesses = new [] { 10, 0, 0, 10 }, Colors = new [] { lightPink, lightBlue, lightBlue, lightPink } },
+				new ExpectedColor { Thicknesses = new [] { 10, 0, 0, 0 }, Colors = new [] { lightPink, lightBlue, lightBlue, lightBlue } },
+				new ExpectedColor { Thicknesses = new [] { 0, 0, 0, 0 }, Colors = new [] { lightBlue, lightBlue, lightBlue, lightBlue } },
+			};
+			var SUT = new BorderCornerRadiusBorderThickness();
+
+			var leftTarget = SUT.LeftTarget;
+			var topTarget = SUT.TopTarget;
+			var rightTarget = SUT.RightTarget;
+			var bottomTarget = SUT.BottomTarget;
+			var centerTarget = SUT.CenterTarget;
+
+			foreach (var expected in expectedColors)
+			{
+				Thickness t = new Thickness(expected.Thicknesses[0], expected.Thicknesses[1], expected.Thicknesses[2], expected.Thicknesses[3]);
+				SUT.Set_MyBorderThickness(t);
+
+				WindowHelper.WindowContent = SUT;
+				WindowHelper.WaitForLoaded(SUT);
+				var renderer = new RenderTargetBitmap();
+				await WindowHelper.WaitForIdle();
+				await renderer.RenderAsync(SUT);
+				var result = await RawBitmap.From(renderer, SUT);
+				WindowHelper.WaitForIdle();
```

Comment:
this chunk here can be extracted into another method

Reply:
https://github.com/unoplatform/uno/pull/10053/files#diff-dee92a7092738201539359d38b4b0b79c497db72f1044a2ddd915d0830bc8f53R192
^ like that

Reply:
I will do it and know it is in my plan to make a PR making that method into a Helper. So it is just temporarily there. :-)